modules/nf-core/vgan/haplocart/main.nf

```diff
@@ -0,0 +1,43 @@
+
+process VGAN_HAPLOCART {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::vgan=1.0.1" : null)
```

Comment:
`enable_conda` has been removed since last tools release, make sure it's in-sync, either by starting from scrach and copy and pasting the relevent vgan changes, or make sure it looks the same with e.g. [a latest module](https://github.com/nf-core/modules/blob/master/modules/nf-core/bwa/mem/main.nf)